docs/developers/contracts/technical-references/tick-ratio.md

```diff
@@ -1,100 +1,96 @@
+Note that ratio and price are in raw numbers, ie, disregarding token decimals. Care must be taken when converting raw prices and ratios to their human readable counterparts with decimals.
+:::
 
-:::danger Important
-All offers in a bin have the same tick. During a [market order](./taking-and-making-offers/taker-order/README.md), **offers in a tick bin are executed in order**, from the first to the last. Inserted offers are **always appended at the end of a tick bin**.
+:::info Remember
+We often casually use the word ’price’ to refer to the ratio between the amount promised by an offer and the amount it requests. In the code however, we use the generic word ’ratio’ to avoid confusion with the normal notion of market price expressed in ’quote’ tokens per ‘base’ token.
 :::
 
 
-### 4. `tickSpacing`
+## Converting a price to a tick
 
-As previously said, each tick points to a single tick bin, where all offers with an identical price are stored. We use a `tickSpacing` value to link the tick bins (i.e. the ticks, i.e. the prices) of an offer list together. If a bin has tick `t`, the following bin has tick `t+tickSpacing`.
+Since the price space is discretized, not all prices can be represented exactly. Converting a price to a tick may therefore require rounding and one should take care to round appropriately; We'll discuss this in detail below.
 
-:::info Note
-* A larger `tickSpacing` means the jumps in tick (i.e. price) are bigger.
-* A smaller `tickSpacing` means the jumps in tick are narrower.
-    * Example: a high-volatility market would require a larger `tickSpacing` compared to a more stable one.
-:::
+First, calculate an exact but possibly non-representable `tick'` which may have a real component:
 
-### 5. Our approach to ticks
+$$
+tick`_{side \in \{asks, bids\}}(price) = \begin{cases}
+   log_{1.0001}(price) &\text{if } side \equiv asks \\
+   -log_{1.0001}(price) &\text{if } side \equiv bids.
+\end{cases}
+$$
 
-Our approach to ticks is inspired from Uniswap’s, with the following notable changes:
-* Directly computing ticks base 1.0001 (not base `sqrt(1.0001)`)
-* Directly computing ratios (not `sqrt(ratio)` - it makes the code simpler when dealing with actual ratios and logs of ratios)
-* Ratios are floating-point numbers, not fixed-point numbers (it increases the precision when computing amounts)
+Next, round this value to an integer to get a representable tick. One will typically want to round down or up depending on whether the price is used in a maker or taker context:
 
-The available prices in an offer list are `1.0001^i` for all `MIN_TICK <= i <= MAX_TICK` such that `i % tickSpacing = 0`.
+| Role           | Tick rounding | Explanation |
+| -------------- | ------------- | ----------- |
+| Taker          | Round down    | Takers specify a max price they're willing to pay. Rounding down the tick corresponds to choosing the tick closest to but not exceeding the specified price. This ensures the taker doesn't pay too much. |
```

Comment:
Takers specify a price, but if buying, they want the lowest price. So it is the max ratio they're willing to pay on the given side. Conversely for makers.

Reply:
Addressed in #203